polyfill/lib/ecmascript.mjs

```diff
@@ -4771,10 +4771,11 @@ export const ES = ObjectAssign({}, ES2022, {
         relativeTo = yearsLater;
         days += monthsWeeksInDays;
 
-        const daysLater = ES.CalendarDateAdd(calendar, relativeTo, { days }, undefined, dateAdd);
+        const wholeDays = new TemporalDuration(0, 0, 0, days);
```

Comment:
should you also "truncate" days before you pass into "new TemporalDuration" here?

Reply:
The reference polyfill implementation is slightly different due to using Number here instead of a mathematical value, so `days` is already an integer in the code.